config/webpack/loaders/devSiteCssFinder.js

```diff
@@ -0,0 +1,20 @@
+const findCss = (x) => {
+  /**
+  * The regex picks out lines that don't contain // but do contain some form of quotation (' or "),
+  * and an explicit .css or .scss deceleration
+  */
+  const regex = /^(?!\/\/|\/).*["'](.*\.css|.*\.scss)["']/m;
+  const m = regex.exec(x);
+  let fileName;
+
+  if (m !== null) {
```

Comment:
could we check if (m && m.length > 0) to quickly assign filename to m[1]?

Reply:
Updated the if to check for (m && m.length > 0) and removed the unnecessary forEach. I had to make the fileName assignment to equal `processedContent[matchIndex]` instead of just `processedContent[1]` because lint was giving me a prefer-destructuring error: [4984d2d](https://github.com/cerner/terra-dev-site/pull/281/commits/4984d2d10c2e10137cfbca58cfbd747cce3e5d83).